tests/src/Traits/MediaCreationTrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online\Traits;
+
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+use Drupal\media\MediaInterface;
+
+/**
+ * Provides methods to create a media from given values.
+ *
+ * This trait is meant to be used only by test classes.
+ */
```

Comment:
Ok for me to split out this trait!

> This trait is meant to be used only by test classes.

I see this note in `Drupal\Tests\media\Traits\MediaTypeCreationTrait`. But other test traits don't have it.
I suspect there is no special reason why the note exists there, probably just a developer feeling a bit different on that day. So I think we can drop it.

Reply:
Done